public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
+  At runtime, Angular discovers the metadata specified by the `@Component`
+  annotation. That's how Angular learns how to do "the right thing".
+
+  The template, metadata, and component together describe a view.
 
-  We apply other metadata decorators in a similar fashion to guide Angular behavior. 
-  The `@Injectable`, `@Input`, `@Output`, `@RouterConfig` are a few of the more popular decorators
+  We apply other metadata !{_decorator}s in a similar fashion to guide Angular behavior.
+  `@Injectable`, `@Input`, `@Output`, and `@RouterConfig` are a few of the more popular !{_decorator}s
   we'll master as our Angular knowledge grows.
 <br clear="all">
 :marked
-  The architectural take-away is that we must add metadata to our code
+  The architectural takeaway is that we must add metadata to our code
   so that Angular knows what to do.
 
 .l-main-section
-<a id="data-binding"></a>
 :marked
-  ## Data Binding
+  ## Data binding
   Without a framework, we would be responsible for pushing data values into the HTML controls and turning user responses
-  into actions and value updates. Writing such push/pull logic by hand is tedious, error-prone and a nightmare to
-  read as the experienced jQuery programmer can attest.
+  into actions and value updates. Writing such push/pull logic by hand is tedious, error-prone, and a nightmare to
+  read as any experienced jQuery programmer can attest.
 figure
   img(src="/resources/images/devguide/architecture/databinding.png" alt="Data Binding" style="width:220px; float:left; margin-left:-40px;margin-right:20px" )
 :marked
-  Angular supports **data binding**, 
+  Angular supports **data binding**,
   a mechanism for coordinating parts of a template with parts of a component.
   We add binding markup to the template HTML to tell Angular how to connect both sides.
-  
-  There are four forms of data binding syntax. Each form has a direction - to the DOM, from the DOM, or in both directions -
+
+  There are four forms of data binding syntax. Each form has a direction &mdash; to the DOM, from the DOM, or in both directions &mdash;
   as indicated by the arrows in the diagram.
 <br clear="all">
 :marked
-  We saw three forms of data binding in our [example](#template) template:
-+makeExample('architecture/ts/app/hero-list.component.1.html', 'binding', 'app/hero-list.component.html (excerpt)')(format=".")
+  We saw three forms of data binding in our [example](#templates) template:
+
++makeExcerpt('app/hero-list.component.1.html', 'binding')
+
 :marked
-  * The `{{hero.name}}` [*interpolation*](displaying-data.html#interpolation) 
-  displays the component's `hero.name` property value within the `<div>` tags.
-  
-  * The `[hero]` [*property binding*](template-syntax.html#property-binding) passes the `selectedHero` from
+  * The `{{hero.name}}` [*interpolation*](displaying-data.html#interpolation)
+  displays the component's `hero.name` property value within the `<li>` tags.
```

Comment:
See my note about this in the code sample. I'm not insisting on reverting this one. But I want to register my distress with the unnecessary HTML bloat introduced by switching from `<div>` to  `<ul><li>`.  

Reasonable minds might differ I suppose.